airflow/hooks/http_hook.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+import requests
+
+from airflow import utils
+from airflow.hooks.base_hook import BaseHook
+
+
+class HttpHook(BaseHook):
+
+    """
+    Interact with HTTP servers.
+    """
+
+    def __init__(
+            self, method='POST', http_conn_id='http_default'):
+        self.http_conn_id = http_conn_id
+        self.method = method
+
+    def get_conn(self, headers):
```

Comment:
I'm thinking about where the `headers` info should live. Maybe in the `extra` section of the connection (when static for a connection) and/or in the Hook's constructor. Or maybe they exist in all these places and one overrides the other....